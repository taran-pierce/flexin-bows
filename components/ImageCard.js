const ImageCard = (props) => {
  return (
    <div>
      <style jsx>{`
        img {
          border: 1px solid rgba(0,0,0,.6);
          max-width: 100%;
        }
      `}</style>
      <img src={props.image} alt={`get it done`} />
    </div>
  )
}

export default ImageCard