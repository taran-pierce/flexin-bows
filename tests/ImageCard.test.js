import React from 'react';
import ImageCard from '../components/ImageCard';
import renderer from 'react-test-renderer';

test('Does ImageCard render correctly?', () => {
  const imgSrc = 'https://placekitten.com/200/300'
  
  const component = renderer.create(
    <ImageCard image={imgSrc} />,
  );
  
  // verify component compiles correctly
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
