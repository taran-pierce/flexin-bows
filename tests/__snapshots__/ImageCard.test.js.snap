// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Does ImageCard render correctly? 1`] = `
<div
  className="jsx-776312227"
>
  <img
    alt="get it done"
    className="jsx-776312227"
    src="https://placekitten.com/200/300"
  />
</div>
`;
